### ISSUE TITLE
style(lint): run yarn format on server & client

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#6314]
predecessor: #6313 + 2 weeks after that PR has been merged

Once #6313 is merged every file will have the same styling, no matter who wrote it.
All that's left to do is run `yarn format` to automatically fix all the files in our codebase.
I want to wait 2 weeks after it is merged to give all the PRs in flight time to use the new styling so there are no merge conflicts.

AC
- [ ] call `yarn format` on the server & submit that PR. If any errors are found (e.g. files that are no longer in use that should be deleted) fix them
- [ ] call `yarn format` on the server & submit a separate PR. If any errors are found (e.g. a reference to a typescript that that does not exist) fix them

Estimate: 5 hours
